app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,83 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+
+  def new
+    @user = User.new
+    render 'devise/registrations/new'
```

Comment:
Let's not override the whole `Devise::Registrations#new` method just to make it render a view from a different folder. Instead, we can just rename the devise generated views from `devise/registrations/new` to `users/registrations/new`, and we can get rid of this method override.